packages/@storefront/flux-capacitor/test/unit/core/requests/utils.ts

```diff
@@ -50,26 +54,28 @@ suite('requests helpers', ({ expect, stub, spy }) => {
       stub(Selectors, 'selectedRefinements').returns(refinements);
       stub(Selectors, 'page').returns(page);
       stub(Selectors, 'pageSize').returns(originalPageSize);
+      stub(RequestHelpers, 'extractSessionId').returns(sessionId);
     });
 
     it('should build out request', () => {
       stub(Selectors, 'config').returns({});
 
-      expect(RequestHelpers.search(<any>{})).to.eql({
+      expect(RequestHelpers.search(state)).to.eql({
         pageSize: remainingRecords,
         area,
         fields,
         query,
         collection,
         refinements,
+        sessionId,
         skip: originalSkip
       });
     });
 
     it('should decrease page size to prevent exceeding MAX_RECORDS', () => {
       stub(Selectors, 'config').returns({ search: {} });
 
-      const { pageSize, skip } = RequestHelpers.search(<any>{});
+      const { pageSize, skip } = RequestHelpers.search(<any> {});
```

Comment:
It pains me to say this, but it looks like our convention is no space between the cast type and the cart target (ie. `<any>{}`.

Reply:
My bad. Updated to remove the space.